data/hai/hai reveal 4 middle.txt

```diff
@@ -0,0 +1,481 @@
+# Copyright (c) 2020 Nomadic Volcano
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+########################################################################
+
+# Part 4: Search Korath and human space for the missing researchers
+
+#Timer trigger runs off Reveal 2 events, and ensures a minimum time between events if you do Reveal 3 events very quickly.
+mission "Hai Rescue [A00] Darkwaste Timer"
+	landing
+	invisible
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai"
+		not attributes "uninhabited"
+		not planet Darkwaste
+	destination Darkwaste
+	to offer
+		"hai slave prereq" >= 3
+		"combat rating" > 3000
+		has "Hai Leaks Response 6: done"
+	on offer
+		event "hai rescue: teeneep message" 20 30
+		fail
+
+
+event "hai rescue: teeneep message"
+
+
+mission "Hai Rescue [A01] Visit Darkwaste"
+	landing
+	name "Missing Researchers"
+	description "Fly to <destination> to visit the Hai you rescued."
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai"
+		not attributes "uninhabited"
+		not planet "Darkwaste"
+	destination Darkwaste
+	to offer
+		has "event: hai rescue: teeneep message"
+		has "Hai Building Mountaintop [3] Furnishings: done"
+		or
+			has "Pirate Troubles [4]: done"
+			has "Pirate Troubles [4]: declined"
+	on offer
+		conversation
+			`You receive an emergency communication from a Hai woman named Teeneep. The message is brief and cryptic, but you deduce that it refers to the Hai you've rescued from human space in the past. They need to talk to you again as soon as possible. There is a set of landing coordinates on the uninhabited Hai planet, <destination>.`
+				accept
+
+mission "Hai Rescue [A02] Microtransceiver Identification"
+	landing
+	name "Missing Researchers"
+	description "Fly to <destination> to ask the Quarg about the microtransceivers."
+	source Darkwaste
+	destination "Kuwaru Efreti"
```

Comment:
Well, on the plus side, there should be a lot more things to go out and discover than just those soon enough.
This whole Kuwaru Efreti bit may well get redone from scratch though.

Reply:
Especially if it's slated for larger rework anyway, I agree it would probably be best to skip sending us to the Efreti.

The Hai or Human Quarg should suffice (if they're even still included at all).